configs/resources/custom_image.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: custom-test-image
+  namespace: redhat-ods-applications
+  annotations:
+    opendatahub.io/notebook-image-name: "Custom Image - Testing Only"
```

Comment:
This file should be included in the Docker image. I don't know if having it in configs/resources/ is the best place  (@bdattoma  might know better).   Maybe it should be in tests/Tests/500__jupyterhub/config

Other suggestions in case you have more custom images for testing in the future:
- rename file to custom_image_elyra.yaml
- Modify the annotation to "Custom Image - Elyra - Testing Only"

Reply:
I don't think we will need more YAML files for this specific functionality, so the name should be fine as is. @bdattoma opinions on where to put the file?

Reply:
hm good question. We don't have a place where to put this kind of files. I generally like to keep them near where they are used. So if this yaml is a resource of jupyter tests, then I'd store it in the jupyter resources folder. The risk is this is not the most tidy approach. 

Maybe we could create a subdirectory under `tests/Resources` called `Files` and store all those kind of resources there. What do you think?

Reply:
Done in latest commit